src/cartesian/Area.tsx

```diff
@@ -49,6 +49,7 @@ export type BaseValue = number | 'dataMin' | 'dataMax';
  */
 interface InternalAreaProps {
   activeDot: ActiveDotType;
+  // TODO: this isn't used
```

Comment:
this param is only used in `Line` - no other used references anywhere in the project (or react-smooth). Probably good to remove before 3.0